rust/src/index/vector/pq.rs

```diff
@@ -229,12 +230,15 @@ impl ProductQuantizer {
 
         let mut codebook_builder =
             Float32Builder::with_capacity(num_centorids as usize * data.value_length() as usize);
+        let rng = rand::rngs::SmallRng::from_entropy();
+
         // TODO: parallel training.
         for sub_vec in &sub_vectors {
             // Centroids for one sub vector.
             let values = sub_vec.values();
             let flatten_array: &Float32Array = as_primitive_array(&values);
-            let centroids = train_kmeans(flatten_array, dimension, num_centorids, 100)?;
+            let centroids =
+                train_kmeans(flatten_array, dimension, num_centorids, 25, rng.clone()).await?;
```

Comment:
Why change to 25?

Reply:
this was the default value in the faiss kmean we used. seems to get good enough results.